Gemfile.lock

```diff
@@ -1,60 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.5.0)
+    better_errors (2.5.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (10.0.2)
+    byebug (11.0.1)
     coderay (1.1.2)
     commander (4.4.7)
       highline (~> 2.0.0)
     database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubi (1.7.1)
-    event_sourcery (0.22.0)
+    erubi (1.9.0)
+    event_sourcery (0.23.0)
     event_sourcery-postgres (0.8.0)
       event_sourcery (>= 0.14.0)
       pg
       sequel (>= 4.38)
-    highline (2.0.0)
+    highline (2.0.3)
     method_source (0.9.2)
     mustermann (1.0.3)
     pg (0.20.0)
     pry (0.12.2)
       coderay (~> 1.1.0)
       method_source (~> 0.9.0)
-    pry-byebug (3.6.0)
-      byebug (~> 10.0)
+    pry-byebug (3.7.0)
+      byebug (~> 11.0)
       pry (~> 0.10)
-    rack (2.0.6)
-    rack-protection (2.0.4)
+    rack (2.0.8)
+    rack-protection (2.0.7)
       rack
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
-    rake (12.3.1)
-    rspec (3.8.0)
-      rspec-core (~> 3.8.0)
-      rspec-expectations (~> 3.8.0)
-      rspec-mocks (~> 3.8.0)
-    rspec-core (3.8.0)
-      rspec-support (~> 3.8.0)
-    rspec-expectations (3.8.2)
+    rake (13.0.1)
+    rspec (3.9.0)
```

Comment:
### [rspec](http://github.com/rspec)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.8.0 → 3.9.0

[_~~change-log~~, [source-code](https://github.com/rspec/rspec)_]





<details>
<summary>Commits</summary>

A change of **2** commits. See the full changes on [the compare page](https://github.com/rspec/rspec/compare/v3.8.0...v3.9.0).

These are the individual commits:
- (cd5ad45) [Merge pull request #33 from orien/project-metadata](https://github.com/rspec/rspec/commit/cd5ad45269ff8058c2e5ecf391a8659e62e1a6e7)
- (be53c57) [Version 3.9.0](https://github.com/rspec/rspec/commit/be53c57fc620117f4c64b2f235bc82d78870198e)


</details>